.circleci/config.yml

```
@@ -1,9 +1,12 @@
 version: 2.1
 
+references:
+  circleci-docker-primary: &circleci-docker-primary trussworks/circleci-docker-primary:99bee5627ff234eb0f31f5899628bff03df78b6d
+
 jobs:
   validate:
     docker:
-      - image: trussworks/circleci-docker-primary:735a60a78114453f050c2ba45400c4aa6c9e06f2
+      - image: *circleci-docker-primary
```

This is new to me. How does it know what the `*` refers too?

---

To be more clear, I'd expect you have to list the owner:

```suggestion
      - image: trussworks/circleci-docker-primary
```

Which would use the latest image.

---

It's a pointer reference to circleci-docker-primary defined up at the top. Dynamike showed me a while ago!

---

Oh that's sweet. I'm going to use this everywhere!